docs/technical-design/file-upload.md

```diff
@@ -0,0 +1,93 @@
+# How FileUpload works on MC-Review
+
+## Introduction
+
+MC-Review allows users to upload and save files through multiple workflows. FileUpload is the main reusable component for file uploads in the application. It supports multiple files at a time via user select or drag and drop.
+
+Currently, the component is embedded inside several larger forms in our application. The document files supported in MC-Review are text based documents (we support  PDF, CSV, DOC, DOCX, XLS, XLSX, XLSM).
```

Comment:
@haworku  Re: "several larger forms" is this trying to get at that we use this component for different features like Q&A and the intake process? I suppose I saw the entirety of MC-Review as a form, but thinking about the future of this app, perhaps it makes sense to refer to the different features and pages as forms since that's how they function.